org.nongnu.lingot.yaml

```diff
@@ -0,0 +1,34 @@
+app-id: org.nongnu.lingot
+runtime: org.gnome.Platform
+runtime-version: "46"
+sdk: org.gnome.Sdk
+
+command: lingot
+finish-args:
+  - --filesystem=home:rw
+  # X11
+  - --share=ipc
+  - --socket=fallback-x11
+  #Wayland
+  - --socket=wayland
+  - --device=dri
+  # Audio
+  - --socket=pulseaudio
```

Comment:
It claims to support JACK so this need to have `--filesystem=xdg-run/pipewire-0` to work